docs/rfc/001-backup-format.md

```diff
@@ -0,0 +1,34 @@
+# Structured Backup format
+
+## Principles
+
+- Based-zip archive format
```

Comment:
How to handle UTF-8? IMO zip doesn't work well with UTF-8 filename.

Reply:
zip64 format